model/execution/DeliveryExecutionManagerService.php

```diff
@@ -285,7 +299,15 @@ public function adjustTimers(array $deliveryExecutions, $seconds)
 
                 $data = $deliveryMonitoringService->getData($deliveryExecution);
                 $data->updateData([DeliveryMonitoringService::REMAINING_TIME]);
+
                 $deliveryMonitoringService->save($data);
+
+                // log adjusted time of the de
+                $adjustmentLog = [
+                    'reason' => $reason,
+                    'increment' => $seconds,
```

Comment:
Not sure if the key `increment` is good. Someone else reading the code might understand that it is only a positive value, while it could be also negative

Reply:
Answered here: https://github.com/oat-sa/extension-tao-act/pull/1365#discussion_r426546078, structure was given to us by customer, for their ODS, not sure that we can change that

Reply:
I guess it means that any value there will be added, even if it is negative value, and formula is value + increment